library/tinkerpop/src/main/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopGraph.java

```diff
@@ -637,12 +637,12 @@ public Iterator<Edge> edges(final Object id, final Direction direction, final St
      * @param ids vertex IDs or edge IDs to be queried for.
      * Supports input as a {@link Vertex}, {@link Edge}, List of Edge IDs or individual Vertex IDs.
      * @param direction {@link Direction} of edges to return.
-     * @param labels    labels of edges. Alternatively you can supply a Gaffer View serialised into JSON.
+     * @param labels labels of edges. Alternatively you can supply a Gaffer View serialised into JSON.
      * @return iterator of {@link GafferPopEdge}s.
      * @see #edges(Object...)
      */
     public Iterator<Edge> edges(final Iterable<Object> ids, final Direction direction, final String... labels) {
-        return edgesWithView(ids, direction, createView(labels));
+        return edgesWithView(getElementSeeds(ids), direction, createView(labels));
```

Comment:
Is this needed? I think `edgesWithView` calls `getElementSeeds` so I think we're doing it twice here